stdlib/public/core/FloatingPoint.swift.gyb

```diff
@@ -1748,12 +2004,55 @@ extension BinaryFloatingPoint {
   ///     let magnitude = x.significand * F.radix ** x.exponent
   public static var radix: Int { return 2 }
 
+  /// Creates a new floating-point value using the sign of one value and the
+  /// magnitude of another.
+  ///
+  /// The following example uses this initializer to create a new `Double`
+  /// instance with the sign of `a` and the magnitude of `b`:
+  ///
+  ///     let a = -21.5
+  ///     let b = 305.15
+  ///     let c = Double(signOf: a, magnitudeOf: b)
+  ///     print(c)
+  ///     // Prints "-305.15"
+  ///
+  /// This initializer implements the IEEE 754 `copysign` operation.
+  ///
+  /// - Parameters:
+  ///   - signOf: A value from which to use the sign. The result of the
+  ///     initializer has the same sign as `signOf`.
+  ///   - magnitudeOf: A value from which to use the magnitude. The result of
+  ///     the initializer has the same magnitude as `magnitudeOf`.
   public init(signOf: Self, magnitudeOf: Self) {
     self.init(sign: signOf.sign,
       exponentBitPattern: magnitudeOf.exponentBitPattern,
       significandBitPattern: magnitudeOf.significandBitPattern)
   }
 
+  /// Returns a Boolean value indicating whether this instance should precede the
+  /// given value in an ascending sort.
+  ///
+  /// This relation is a refinement of the less-than-or-equal-to operator
+  /// (`<=`) that provides a total order on all values of the type, including
+  /// noncanonical encodings, signed zeros, and NaNs. Because it is used much
+  /// less frequently than the usual comparisons, there is no operator form of
+  /// this relation.
```

Comment:
That's a good point. I think this language might have originated in the floating-point protocols proposal, where it made more sense. I'll look at removing it.